traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -1424,6 +1426,94 @@ sub drop_qstring_dot_config {
 	return $text;
 }
 
+sub logging_dot_yaml {
+	my $self        = shift;
+	my $profile_obj = shift;
+
+	my $data = $self->profile_param_data( $profile_obj->id, "logging.yaml" );
+
+	# This is an YAML file, so we need to massage the header a bit for YAML commenting.
+	my $text = "# " . $self->header_comment( $profile_obj->name );
+	$text =~ s/# //;
+	$text =~ s/\n//;
```

Comment:
We appear to be adding a `# ` and then removing it.

Reply:
Removed the # stripping and the # usage in the comment. The default header comment has a # in it and so is already a valid YAML comment